iis-workflows/iis-workflows-core-examples/src/test/java/eu/dnetlib/iis/core/examples/spark/PersonIdExtractorWorkflowTest.java

```diff
@@ -0,0 +1,34 @@
+package eu.dnetlib.iis.core.examples.spark;
+
+import org.junit.Test;
+
+import eu.dnetlib.iis.core.AbstractOozieWorkflowTestCase;
+import eu.dnetlib.iis.core.OozieWorkflowTestConfiguration;
+import eu.dnetlib.iis.core.TestsIOUtils;
+import eu.dnetlib.iis.core.WorkflowTestResult;
+
+/**
+ * 
+ * @author madryk
+ *
+ */
+public class PersonIdExtractorWorkflowTest extends AbstractOozieWorkflowTestCase {
```

Comment:
I've added `@Category` for `AbstractOozieWorkflowTestCase`. Earlier it was not annotated at all.